Authentication/CloudPakForData/CloudPakForDataAuthenticator.cs

```diff
@@ -146,17 +232,29 @@ private bool RequestToken(Callback<CloudPakForDataTokenResponse> callback)
                 throw new ArgumentNullException("successCallback");
 
             RESTConnector connector = new RESTConnector();
-            connector.URL = Url;
+            connector.URL = Url + UrlSuffix;
```

Comment:
The url suffix should not be added nakedly as this would lead to suffix duplication if the doc examples were followed

Reply:
This has already been addressed